package.json

```diff
@@ -7,22 +7,12 @@
     "test": "test"
   },
   "dependencies": {
-    "babel-polyfill": "^6.26.0",
-    "classnames": "^2.2.5",
-    "lodash": "^4.17.11",
-    "patternfly": "^3.58.0",
-    "patternfly-react": "^2.25.4",
-    "prop-types": "^15.6.2",
-    "react": "^16.6.3",
-    "react-bootstrap": "^0.32.1",
-    "react-dom": "^16.6.3",
-    "react-json-tree": "^0.11.0",
-    "reselect": "^3.0.1",
-    "seamless-immutable": "^7.1.3"
+    "@theforeman/vendor": "^1.4.0",
```

Comment:
Does this mean we have no version of ansible plugin that works with foreman 1.23?

Reply:
This is the original PR where I tried to add the vendor from the beginning. Because it didn't merge yet, I updated to use the latest vendor.

Would you suggest creating 2 different PRs for `1.23` and `1.24`?

Reply:
Yes. We need a version that is compatible with 1.23, otherwise users won't be able to upgrade.